### PR TITLE
PROD - build fixes and minor Node version changes to support latest pg npm package, also PM-61

### PR DESCRIPTION
**Pull request by jmgasper**
1. Build fixes for deployment to address latest failures
2. Updated the `pg` npm version in package.json to better support the consolidated Postgres DB in RDS
3. Change to use a different docker image for Node, since 8.2 had some odd interactions with the new `pg` npm version.  8.17 does not have the same problems.
4. Updated to use v5 member API instead of v3, for invitations by handle (https://topcoder.atlassian.net/browse/PM-61)

### CODE REVIEW

**Review by testflyjets — APPROVED** [review]

File: connect/service.js [hunk 1]
```diff
@@ -128,7 +128,13 @@ const getRoleMembers = (role) => (
  * @return {Promise}   resolves to the list of user details
  */
 const getUsersById = (ids) => {
-  const query = _.map(ids, (id) => 'userId:' + id).join(' OR ');
+
+  let query = ""
+  if (ids.length>1) {
```
Comment (testflyjets) [inline connect/service.js:133]: Correct me on this, but I thought our coding standard for JS would write this as:
```
if (ids.length > 1) {
```
Let me know if that's off base.

File: connect/service.js [hunk 2]
```diff
@@ -128,7 +128,13 @@ const getRoleMembers = (role) => (
...
+    query = _.map(ids, (id) => 'userIds=' + id).join('&');
+  } else if(ids.length==1) {
```
Comment (testflyjets) [inline connect/service.js:135]: Same comment as above regarding spaces around operators.

File: connect/service.js [hunk 3]
```diff
@@ -137,19 +143,17 @@ const getUsersById = (ids) => {
     .then((token) => {
       const fields = 'fields=userId,email,handle,firstName,lastName,photoURL,status';
       return request
-      .get(`${config.TC_API_V3_BASE_URL}/members/_search?${fields}&query=${query}`)
+      .get(`${config.TC_API_V5_BASE_URL}/members/?${fields}&${query}`)
       .set('accept', 'application/json')
       .set('authorization', `Bearer ${token}`)
       .then((res) => {
-        if (!_.get(res, 'body.result.success')) {
+        if (res.status!=200) {
```
Comment (testflyjets) [inline connect/service.js:150]: Spaces around operators.

**Review by testflyjets — APPROVED** [review]
Minor formatting question.

File: connect/service.js (hunk 2 above)
Comment (jmgasper) [reply connect/service.js]: Fixed

File: connect/service.js (hunk 3 above)
Comment (jmgasper) [reply connect/service.js]: Fixed

**Pull request merged**